tests/Functional/Admin/SharedBlockAdminTest.php

```diff
@@ -45,11 +45,151 @@ public function testCrudUrls(string $url, array $parameters = []): void
     public static function provideCrudUrlsCases(): iterable
     {
         yield 'List Block' => ['/admin/tests/app/sonatapageblock/shared/list'];
-        yield 'Create Block' => ['/admin/tests/app/sonatapageblock/shared/create'];
+        yield 'List Block types' => ['/admin/tests/app/sonatapageblock/shared/create'];
+
+        yield 'Create Block' => ['/admin/tests/app/sonatapageblock/shared/create', [
+            'type' => 'sonata.page.block.shared_block',
+        ]];
+
         yield 'Edit Block' => ['/admin/tests/app/sonatapageblock/shared/1/edit'];
         yield 'Remove Block' => ['/admin/tests/app/sonatapageblock/shared/1/delete'];
     }
 
+    /**
+     * @dataProvider provideFormUrlsCases
+     *
+     * @param array<string, mixed> $parameters
+     * @param array<string, mixed> $fieldValues
+     */
+    public function testFormsUrls(string $url, array $parameters, string $button, array $fieldValues = []): void
+    {
+        $client = self::createClient();
+
+        $this->prepareData();
+
+        $client->request('GET', $url, $parameters);
+        $client->submitForm($button, $fieldValues);
+        $client->followRedirect();
+
+        self::assertResponseIsSuccessful();
+    }
+
+    /**
+     * @return iterable<array<string|array<string, mixed>>>
+     *
+     * @phpstan-return iterable<array{0: string, 1: array<string, mixed>, 2: string, 3?: array<string, mixed>}>
+     */
+    public static function provideFormUrlsCases(): iterable
+    {
+        // Blocks From SonataBlockBundle
+        // yield 'Create Shared Block - Container' => ['/admin/tests/app/sonatapageblock/shared/create', [
```

Comment:
This one is broken on SonataBlockBundle.

Reply:
Can it be fix ?

Reply:
Yes, we need to remove this field: https://github.com/sonata-project/SonataBlockBundle/blob/97cc8c77d4c87203e2cfd052c0853ef932239955/src/Block/Service/ContainerBlockService.php#L37

Reply:
And this PR: https://github.com/sonata-project/SonataBlockBundle/pull/1098